libs/shared/ui/src/lib/components/tag-commit/tag-commit.tsx

```diff
@@ -26,13 +27,15 @@ export function TagCommit(props: TagCommitProps) {
 
   useEffect(() => {
     return () => clearTimeout(displayCopy)
-  }, [])
+  }, [displayCopy])
```

Comment:
What is this? It seems like a hack why did you do that?

Reply:
I don't know, this code was already here, I just added the missing dependency